tests/testthat/_snaps/theme/custom-strip-elements-can-render.svg

```diff
@@ -18,7 +18,7 @@
   </clipPath>
 </defs>
 <g clip-path='url(#cpMC4wMHw3MjAuMDB8MC4wMHw1NzYuMDA=)'>
-<rect x='0.000000000000064' y='0.00' width='720.00' height='576.00' style='stroke-width: 1.07; stroke: #FFFFFF; fill: #FFFFFF;' />
+<rect x='0.00' y='0.00' width='720.00' height='576.00' style='stroke-width: 1.07; stroke: #FFFFFF; fill: #FFFFFF;' />
```

Comment:
I don't think this snapshot is snapshotting what it should. I think a `registerS3method()` call is missing from the test, because it renders rectangles interactively, but you don't see this back in the snapshot.